manual/debugging/file-checks-do-not-run.md

```diff
@@ -0,0 +1,65 @@
+# Debug failing file checks
+
+Symptoms: the file check progress get stuck on 0%:
+
+![](./images/file-checks-empty-progress-bars.png)
+
+Or the end-to-end tests fail at the point where the user should be redirected to
+the file checks result page.
+
+## What to check
```

Comment:
@MancunianSam Do you have any other tricks for debugging the file checks?

Reply:
Sometimes I will run the lambdas using test data through the AWS console, especially if I suspect it's the EFS problem but I appreciate this is probably not best practice so you don't have to add that :smile: 
Otherwise, I can't think of anything you haven't covered.

Reply:
On integration, that doesn't seem too bad. 🙂 

I think we'll need to implement [TDR-94](https://national-archives.atlassian.net/browse/TDR-94) at some point soon, and that will give us the ability to kick off file checks in any environment.